moviebot/nlu/annotation/semantic_annotation.py

```diff
@@ -71,7 +77,7 @@ def from_span(
                 AnnotationType.NAMED_ENTITY. Defaults to None.
 
         Returns:
-            [type]: [description]
+            New semantic annotation retaining span information.
```

Comment:
Missing return type to the method signature. 
In the docstring:
- `Create` -> `Creates`
- The type of the arguments can be removed
- Add missing dots

Reply:
Done